docs/task-cluster.md

```diff
@@ -96,3 +96,23 @@ tasks:
             dataset-category: train
             stage: merge-corpus
 ```
+
+## Interactive Tasks
+
+Taskcluster allows authorized users to run so-called [interactive tasks](https://docs.taskcluster.net/docs/reference/workers/docker-worker/features#feature-interactive). These tasks allow users to gain a shell in the same environment that a pipeline step runs in. This can often be useful for quicker debugging or testing of ideas.
+
+To start an interactive task, follow these steps:
+
+1. Go to the task you want an interactive version of, eg: https://firefox-ci-tc.services.mozilla.com/tasks/DZvVQ-VUTPSyPBBS13Bwfg
```

Comment:
Something I always ask myself when using an interactive task: is the "command" in the payload just ignored?

Reply:
Today I learned that they don't :). I'll update these docs to account for that.